client/src/views/FollowList/index.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div>
+    <q-tabs
+      v-model="tab"
+      align="justify"
+      active-color="primary"
+      indicator-color="primary"
+      narrow-indicator
+    >
+      <q-tab class="text-orange" name="followers" label="followers" />
+      <q-tab class="text-teal" name="following" label="following" />
+    </q-tabs>
+    <q-tab-panels
+      v-model="tab"
+      animated
+      transition-prev="scale"
+      transition-next="scale"
+    >
+      <q-tab-panel name="followers">
+        <followItem
+          :follow="followersGetter"
+        />
+        <p v-if="!followersGetter.length" class="notification">
+          no followers found...
+        </p>
+      </q-tab-panel>
+
+      <q-tab-panel name="following">
+        <followItem
+          :follow="followingGetter"
+        />
+        <p v-if="!followingGetter.length" class="notification">
+          no following found...
+        </p>
+      </q-tab-panel>
+    </q-tab-panels>
+  </div>
+</template>
+
+<script>
+import { mapActions, mapGetters } from 'vuex'
+import FollowItem from './FollowItem'
+export default {
+  methods: mapActions(['uploadProfile']),
+  computed: mapGetters(['followingGetter', 'followersGetter']),
```

Comment:
Please follow proper ordering convention(makes this easy to read/navigate through):
props > data > computed > lifecycle callbacks > methods